src/github/nag.rs

```diff
@@ -700,15 +700,17 @@ fn cancel_fcp(author: &GitHubUser, issue: &Issue, existing: &FcpProposal) -> Das
     // leave github comment stating that FCP proposal cancelled
     let comment = RfcBotComment::new(issue, CommentType::FcpProposalCancelled(author));
     let _ = comment.post(None);
-    &[
```

Comment:
Not sure why Clippy suggests replacing the `for` loop here? I think it was more readable as-is.

Reply:
Done.